docs/src/add-services/redis.md

```diff
@@ -4,16 +4,12 @@ weight: 11
 sidebarTitle: "Redis"
 ---
 
-Redis is a high-performance in-memory object store, well-suited for application level caching.
-
-See the [Redis documentation](https://redis.io/documentation) for more information.
-
+[Redis](https://redis.io/documentation) is a multi-model database that allows you to store data in memory
+for high-performance data retrieval and storage purposes.
```

Comment:
:hammer_and_wrench: Maybe I'm too allergic to the word `purposes`, but I think it works without it. Otherwise, it is clearer now.
```suggestion
for high-performance data retrieval and storage.
```